go/vt/vtgate/planbuilder/show.go

```diff
@@ -167,10 +167,11 @@ func buildShowTableStatusPlan(show *sqlparser.ShowTableStatus, vschema ContextVS
 	// Remove Database Name from the query.
 	show.DatabaseName = ""
 
+	query := sqlparser.String(show)
 	return &engine.Send{
 		Keyspace:          keyspace,
 		TargetDestination: destination,
-		Query:             sqlparser.String(show),
+		Query:             query,
```

Comment:
looks like this was change got added to PR which is not related to workload changes.

Reply:
This was something GoLand recommeded to do. Do you want me to revert it back as it was before.